src/Razor/benchmarks/Microsoft.AspNetCore.Razor.Microbenchmarks/Resources/project.razor.json

```diff
@@ -1,10 +1,11 @@
 {
-  "__Version": 5,
+  "__Version": 6,
   "ProjectKey": "C:\\Users\\admin\\location\\blazorserver\\obj\\Debug\\net7.0\\",
   "FilePath": "C:\\Users\\admin\\location\\blazorserver\\blazorserver.csproj",
   "Configuration": {
     "ConfigurationName": "MVC-3.0",
     "LanguageVersion": "3.0",
+    "SuppressAddComponentParameter": false,
```

Comment:
See my suggestions in the `ObjectReaders.cs` and `ObjectWriters.cs`, but you can avoid writing these properties if they're set to their default values. Then, you don't need to add them here.